### ISSUE TITLE
Saved movies in both .avi & .mp4 while set to HEVC (.mp4)

### ISSUE THREAD

**Issue by NginUS** [ccrisan/motioneye#1283]
I recently changed the settings in Motioneye to save as HEVC (.mp4) from having it set previously as MPEG4 (.avi).

When I checked tonight I see it's recording in both formats, & it doesn't appear to be completely consistent. Sometimes its saved as both for the same timestamp & other times not.

Screenshot:

![2x](https://user-images.githubusercontent.com/8769067/57977665-1c33f580-79cb-11e9-9d64-f42b890b67ad.png)

**Comment by NginUS** [comment]
Is there a config file somewhere that might be wrongly causing duplicates to be saved like this? This is a significant issue for me- I don't have the kind of disk space to support this behavior. Would really appreciate a solution for this.

**Comment by ilovelinux** [comment]
Hey @NginUS, still working on this problem? Can you send us your camera's config file? You can find it on `/etc/motioneye/name_of_the_camera.conf`

**Comment by NginUS** [comment]
No, the camera was one of those cheap ones from China, & died soon after this thread was started. The app's been uninstalled since so I'm afraid I can't help here anymore.
